ecosystem/typescript/sdk/package.json

```diff
@@ -53,13 +54,14 @@
     "eslint-config-airbnb-typescript": "^17.0.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-import": "^2.25.4",
-    "jest": "^29.0.0",
-    "openapi-typescript-codegen": "^0.23.0",
+    "jest": "^28.1.3",
+    "openapi-typescript-codegen": "https://github.com/aptos-labs/openapi-typescript-codegen/releases/download/v0.23.0/openapi-typescript-codegen-v0.23.0.tgz",
```

Comment:
Do we have to hardcode this?

Reply:
This is a forked version, not from the public registry.